Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -452,58 +452,68 @@ void Painter::draw_bitmap(const IntPoint& p, const GlyphBitmap& bitmap, Color co
     }
 }
 
-void Painter::draw_triangle(const IntPoint& a, const IntPoint& b, const IntPoint& c, Color color)
+void Painter::draw_triangle(const IntPoint& a, const IntPoint& b, const IntPoint& c, const Color& color)
```

Comment:
You can just pass `Color` by value, it's only a 32 bit value. I believe we pass it by value almost everywhere.